app/components/hero/hero.html

```diff
@@ -1,4 +1,14 @@
 <section class="hero--component">
+  <div>
+    <a href="{{$ctrl.hero.githubRepo}}" target="_blank" class="github-corner" aria-label="Fork me!">
+      <svg width="80" height="80" viewBox="0 0 250 250" class="github-button" aria-hidden="true">
```

Comment:
While this is very cute, it does violate the GitHub brand guidelines: https://github.com/logos